spyder/plugins/ipythonconsole.py

```diff
@@ -1079,11 +1099,38 @@ def create_new_client(self, give_focus=True, filename='', is_cython=False):
                                      "<tt>conda install ipykernel cloudpickle</tt>"))
                 return
 
-        self.connect_client_to_kernel(client, is_cython=is_cython)
+        self.connect_client_to_kernel(client, is_cython=is_cython,
+                                      is_pylab=is_pylab, is_sympy=is_sympy)
         if client.shellwidget.kernel_manager is None:
             return
         self.register_client(client)
 
+    def create_pylab_client(self, give_focus=True, filename='',
+                            is_cython=False, is_pylab=True, is_sympy=False):

```

Comment:
`is_pylab=True` is wrong. It should `is_pylab=False`, as the others.

Reply:
Sorry, silly me. It's right.

But we certainly don't need all these args here. This should be a method without args because their values can be passed directly to `self.create_new_client` inside it.